nalgebra-sparse/example/spmm.rs

```diff
@@ -0,0 +1,36 @@
+extern crate nalgebra_sparse;
```

Comment:
I think we should remove this whole example before merging, because it's not an actual *example* that shows how to use the library, it's a benchmark. And I guess since @losanc is working on benchmarks we'll eventually get a nice benchmark suite that will fill this role.